lib/puppet/provider/ios_stp_global/command.yaml

```diff
@@ -1,84 +1,125 @@
 ---
 get_values:
   default: 'show running-config | begin spanning-tree'
+get_bridge_assurance:
+  default: 'show spanning-tree summary'
 attributes:
   name:
     default:
       default: 'default'
       get_value: 'default'
   bridge_assurance:
     exclusions:
+    - device: '3650'
     - device: '3750'
     - device: '2960'
+    - device: '4503'
     - device: '4507'
     - device: '4948'
     default:
-      get_value: 'no spanning-tree bridge assurance'
+      get_value: '(?:Bridge Assurance\s*is (enabled|disabled))'
       set_value: 'spanning-tree bridge assurance'
       unset_value: 'no spanning-tree bridge assurance'
+      can_have_no_match: 'true'
+  extend_system_id:
+    default:
+      get_value: '(?:spanning-tree extend )(?<system-id>\S*)\\n'
+      set_value: 'spanning-tree extend system-id'
+      unset_value: 'no spanning-tree extend system-id'
```

Comment:
These unset values may cause issues from first look - running tests to verify atm; as in the provider it has `'no ' + command` which to me would be a double `no`

Reply:
good call, check in the delete that it isn't already a 'no ' command